src/ecalc/libraries/libecalc/common/tests/core/consumers/system/conftest.py

```diff
@@ -203,6 +203,8 @@ def compressor_model_result() -> CompressorTrainResult:
         ],
         rate_sm3_day=[np.nan, np.nan, np.nan],
         failure_status=[None, None, None],
+        requested_inlet_pressure=[np.nan, np.nan, np.nan],
```

Comment:
Why set to NaN? Wouldnt it be better with real numbers?

Reply:
As it is only a test, not using the requested in- and outlet pressures to anything - I figured it was ok just to add NaN. No problem to add real numbers though.